devicetypes/smartthings/zwave-water-valve.src/zwave-water-valve.groovy

```diff
@@ -59,18 +59,22 @@ def updated() {
 
 def parse(String description) {
 	log.trace "parse description : $description"
-	def result = null
 	def cmd = zwave.parse(description, [0x20: 1])
 	if (cmd) {
-		result = createEvent(zwaveEvent(cmd))
+		def events = zwaveEvent(cmd)
+		def evt1 = createEvent(events[0])
+		log.debug "Parse returned ${evt1?.descriptionText}"
+		def evt2 = createEvent(events[1])
+		log.debug "Parse returned ${evt2?.descriptionText}"
 	}
-	log.debug "Parse returned ${result?.descriptionText}"
-	return result
+	return [evt1, evt2]
```

Comment:
That's how I had it before. I was following the example outlined here: http://docs.smartthings.com/en/latest/ref-docs/device-handler-ref.html?highlight=createEvent#createevent

@workingmonk, can you clarify what you mean by pointing me at an example maybe?